website/source/docs/providers/openstack/r/networking_port_v2.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "openstack"
+page_title: "OpenStack: openstack_networking_port_v2"
+sidebar_current: "docs-openstack-resource-networking-port-v2"
+description: |-
+  Manages a V2 port resource within OpenStack.
+---
+
+# openstack\_networking\_port_v2
+
+Manages a V2 port resource within OpenStack.
+
+## Example Usage
+
+```
+resource "openstack_networking_network_v2" "network_1" {
+  name = "network_1"
+  admin_state_up = "true"
+}
+
+resource "openstack_networking_port_v2" "port_1" {
+  name = "port_1"
+  network_id = "${openstack_networking_network_v2.network_1.id}"
+  admin_state_up = "true"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Required) The region in which to obtain the V2 networking client.
+    A networking client is needed to create a port. If omitted, the
+    `OS_REGION_NAME` environment variable is used. Changing this creates a new
+    port.
+
+* `name` - (Optional) A unique name for the port. Changing this
+    updates the `name` of an existing port.
+
+* `network_id` - (Required) The ID of the network to attach the port to. Changing
+    this creates a new port.
+
+* `admin_state_up` - (Optional) Administrative up/down status for the port
+    (must be "true" or "false" if provided). Changing this updates the
+    `admin_state_up` of an existing port.
+
+* `mac_address` - (Optional) Specify a specific MAC address for the port. Changing
+    this creates a new port.
+
+* `tenant_id` - (Optional) The owner of the Port. Required if admin wants
+    to create a port for another tenant. Changing this creates a new port.
+
+* `device_owner` - (Optional) The device owner of the Port. Changing this creates
+    a new port.
+
+* `security_groups` - (Optional) A list of security groups to apply to the port.
```

Comment:
Yes, this should be updated to more clearly reflect what is mentioned in the body of the text. I've made a note to open an issue when I have a second - or if you wanted to open one, that'd be great :smile: